src/test/java/taskFinal/page/WomenPage.java

```diff
@@ -0,0 +1,104 @@
+package taskFinal.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import taskFinal.util.Product;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WomenPage {
+
+    private WebDriver driver;
+
+    @FindBy(xpath = "//a[text()='Tops']")
+    WebElement topsLink;
+
+    @FindBy(css = "ol.products")
+    WebElement productItems;
+
+    @FindBy(xpath="//a[text()='shopping cart']")
+    WebElement viewShoppingCartLink;
+
+    @FindBy(xpath = "//a[text()='shopping cart']/..")
+    WebElement addedProductLabel;
+
+    @FindBy(css = "span.counter-number")
+    WebElement counterNumberLabel;
+
+    public WomenPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void getTopsCatalog()
+    {
+        topsLink.click();
+    }
+
+    public List<Product> addProductItems()
+    {
+        driver.manage().window().fullscreen();
```

Comment:
Driver configuration should be in one class. The best place is SingleWebDriver.

Reply:
I have to leave this line, because when driver reaches women page the screen size is reduced